src/rfc3161_client/__init__.py

```diff
@@ -1,3 +1,38 @@
-"""RFC3161 Client."""
+"""rfc3161-client"""
+
+from .base import decode_timestamp_response
+from .errors import VerificationError
+from .tsp import (
+    Accuracy,
+    MessageImprint,
+    PKIStatus,
+    SignedData,
+    SignerInfo,
+    TimeStampRequest,
+    TimeStampResponse,
+    TimeStampTokenInfo,
+)
+from .verify import VerifyOpts, create_verify_opts, verify_signed_data, verify_timestamp_response
+
+__all__ = [
+    # From base.py
+    "decode_timestamp_response",
+    # From verify.py
+    "verify_signed_data",
+    "verify_timestamp_response",
+    "create_verify_opts",
+    "VerifyOpts",
+    # From errors.py
+    "VerificationError",
+    # From tsp.py
```

Comment:
(removed the comments, since the import statements already have that info)
```suggestion
    "decode_timestamp_response",
    "verify_signed_data",
    "verify_timestamp_response",
    "create_verify_opts",
    "VerifyOpts",
    "VerificationError",
```